libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -365,13 +344,8 @@ def __build_clang_tidy_config_handler(args, context):
         check_env['PATH'] = os.path.dirname(config_handler.analyzer_binary)
     clang_bin = analyzer_clangsa.ClangSA.resolve_missing_binary('clang',
                                                                 check_env)
-    if os.path.isfile(clang_bin):
```

Comment:
Resolve missing binary should always work. We should not fallback to anything else but fix resolve missing binary if it is not correct for a use case.